src/clammit/scan_interceptor.go

```diff
@@ -113,8 +115,14 @@ func (c *ScanInterceptor) respondOnVirus(w http.ResponseWriter, filename string,
 		http.Error(w, "Internal Server Error", 500)
 		return true
 	} else if hasVirus {
+		w.Header().Set("Content-Type", c.VirusResponseContentType)
 		w.WriteHeader(c.VirusStatusCode)
-		w.Write([]byte(fmt.Sprintf("File %s has a virus!", filename)))
+
+		if len(c.VirusResponseBody) > 0 {
+			w.Write([]byte(c.VirusResponseBody))
+		} else {
+			w.Write([]byte(fmt.Sprintf("File %s has a virus!", filename)))
+		}
```

Comment:
Yes we are losing the `filename` in the response - however it's not a big loss :-)